landoui/landoapi.py

```diff
@@ -69,6 +69,15 @@ def request(self, method, url_path, *, require_auth0=False, **kwargs):
             response = self.session.request(
                 method, self.url + url_path, **kwargs
             )
+
+            logger.debug(
+                'lando-api response',
+                extra={
+                    'status code': response.status_code,
+                    'content type': response.headers.get('Content-Type'),
+                }
```

Comment:
Convention throughout Lando API/UI is to use `_` as the word separator in these logging calls.